website/src/pages/index.js

```diff
@@ -14,269 +17,144 @@ function getBanner() {
-                  </p>
-                </div>
-                <div className="card__footer">
-                  <Link
-                    className="button button--primary"
-                    to="/docs/introduction">
-                    Learn More
-                  </Link>
-                </div>
-              </div>
-            </div>
-            <div className="col col--4">
-              <div className="card large dark darker">
-                <div className="card__header">
-                  <h3>Getting started guide</h3>
-                </div>
-                <div className="card__body">
-                  <p>
-                    Read this guide to learn how to build, test, and deploy a new dbt project.
-                  </p>
-                </div>
-                <div className="card__footer">
-                  <Link
-                    className="button button--primary"
-                    to={useBaseUrl('guides/getting-started')}>
-                    Learn More
-                  </Link>
-                </div>
-              </div>
-            </div>
-            <div className="col col--4">
-              <div className="card large dark darkest">
-                <div className="card__header">
-                  <h3>dbt Learn</h3>
-                </div>
-                <div className="card__body">
-                  <p>
-                    Learn dbt on your own time with our on demand course or sign up for an upcoming, live public course.
-                  </p>
-                </div>
-                <div className="card__footer">
-                  <Link
-                    className="button button--primary"
-                    to="https://learn.getdbt.com">
-                    Learn Now
-                  </Link>
+        <div className="container container--fluid home" style={{ "padding": "0", "background": "#FFF" }}>
+          <Hero heading="Welcome to the dbt Developer Hub" subheading="Your home base for learning dbt, connecting with the community and contributing to the craft of analytics engineering " showGraphic />
+          <section className="resource-section row">
+            <div className="popular-header"><h2>Popular resources</h2></div>
+            <div className="popular-resources">
+              <div className="grid">
+                <div>
+                  <Card
+                    title="What is dbt?"
+                    body="dbt enables data practitioners to adopt software engineering best practices and deploy modular, reliable analytics code."
+                    link="/docs/introduction"
+                    icon="question-mark"
+                  />
+                </div>
+                <div>
+                  <Card
+                    title="Getting started guide"
+                    body="Learn how to set up dbt and build your first models. You will also test and document your project, and schedule a job."
+                    link="/guides/getting-started"
```

Comment:
todo: I'll work with @runleonarun to determine where this link should be pointed with the move of getting started guide from Guides to docs

Reply:
Sounds good, I'll add this as a todo item before merge!